doc/source/changelog.rst

```diff
@@ -9,5 +9,63 @@ This document contains the release notes for the project.
 
 .. towncrier release notes start
 
+`0.68.2 <https://github.com/ansys/pymapdl/releases/tag/v0.68.2>`_ - 2024-06-18
+==============================================================================
+
+Added
+^^^^^
+
+- refactor: refactoring CLI `#2960 <https://github.com/ansys/pymapdl/pull/2960>`_
```

Comment:
doesn't it make more sense to have ``refactor`` in ``Changed`` section?

Reply:
@clatapie

Reply:
Yes we can do it this way. As there were no [labels](https://github.com/ansys/pymapdl/pull/2960), I chose but we can modify it.